typescript/infra/scripts/funding/calculate-relayer-daily-burn.ts

```diff
@@ -0,0 +1,139 @@
+import postgres, { Sql } from 'postgres';
+
+import { ChainMap } from '@hyperlane-xyz/sdk';
+import { rootLogger } from '@hyperlane-xyz/utils';
+
+import rawDailyRelayerBurn from '../../config/environments/mainnet3/balances/dailyRelayerBurn.json';
+import { mainnet3SupportedChainNames } from '../../config/environments/mainnet3/supportedChainNames.js';
+import rawTokenPrices from '../../config/environments/mainnet3/tokenPrices.json';
+import { fetchLatestGCPSecret } from '../../src/utils/gcloud.js';
+import { writeJsonAtPath } from '../../src/utils/utils.js';
+
+import { RELAYER_MIN_DOLLAR_BALANCE_PER_DAY } from './utils/constants.js';
+import { formatDailyRelayerBurn } from './utils/grafana.js';
+
+const tokenPrices: ChainMap<string> = rawTokenPrices;
+const currentDailyRelayerBurn: ChainMap<number> = rawDailyRelayerBurn;
+
+const DAILY_BURN_PATH =
+  './config/environments/mainnet3/balances/dailyRelayerBurn.json';
+
+const LOOK_BACK_DAYS = 10; // the number of days to look back for average destination tx costs
+const MIN_NUMBER_OF_TXS = 200; // the minimum number of txs to consider for daily burn
+
+async function main() {
+  const chainsMissingInTokenPrices = mainnet3SupportedChainNames.filter(
+    (chain) => !(chain in tokenPrices),
+  );
+
+  if (chainsMissingInTokenPrices.length > 0) {
+    rootLogger.error(
+      `Token prices missing for chains: ${chainsMissingInTokenPrices.join(
+        ', ',
+      )} consider adding them to tokenPrices.json and running the script again.`,
+
+      process.exit(1),
+    );
```

Comment:
think you meant to exit after the log
```suggestion
    );
    process.exit(1);
```